content/login/login.html

```diff
@@ -54,33 +54,176 @@
     </div>
```

Comment:
Almost everything looks good.  I like all the wording we have.  There is some image strangeness that I think we should clean up.  Take a look a screenshot, what can we do about that?

![2017-06-28-215820_1523x772_scrot](https://user-images.githubusercontent.com/5056218/27657433-f0d6c662-5c4c-11e7-9bf2-7b230d000bc9.png)